Sources/tuist/tuist.docc/Articles/Tuist Cloud/Tuist Cloud - Binary caching.md

```diff
@@ -45,7 +45,7 @@ After creating the project, modify your `Tuist/Config.swift` file to reference t
 ```swift
 import ProjectDescription
 
-let config = Config(cloud: .cloud(projectId: "my-organization/my-project"))
+let config = Config(cloud: .cloud(projectId: "my-organization/my-project", url: "https://cloud.tuist.io"))
```

Comment:
isn't this PR to avoid to have to specify it? I wouldn't put it in the example, which they might just copy-paste
```suggestion
let config = Config(cloud: .cloud(projectId: "my-organization/my-project"))
```